tests/test_octodns_provider_cloudflare.py

```diff
@@ -160,9 +160,16 @@ def test_apply(self):
                 'name': 'under.unit.tests',
                 'ttl': 3600
             }),
-        ])
+            # make sure semicolons are not escaped when sending data
+            call('POST', '/zones/42/dns_records', data={
+                'content': 'v=DKIM1;k=rsa;s=email;h=sha256;p=A/kinda+of/long/string+with+numb3rs',
```

Comment:
This line fails the checks in `./script/lint` (which also just made me realize that I forgot to setup CI on this repo :frowning_face:, looking at that now.

```
sweetums:octodns ross$ ./script/lint
tests/test_octodns_provider_cloudflare.py:165:80: E501 line too long (98 > 79 characters)
```